config/default/core.extension.yml

```diff
@@ -26,6 +26,7 @@ module:
   chosen: 0
   chosen_lib: 0
   ckeditor: 0
```

Comment:
Disable the ckeditor 4 module and all dependent modules

Reply:
Done. Removed ckeditor 4 and related modules. Also updated or removed custom code and configuration that used this module.